apps/ledger-live-mobile/src/newArch/features/Accounts/screens/AddAccount/components/SelectAddAccountMethod/useSelectAddAccountMethodViewModel.ts

```diff
@@ -55,7 +56,10 @@ const useSelectAddAccountMethodViewModel = ({
   const onClickAdd = useCallback(() => {
     trackButtonClick("With your Ledger");
     onClose?.();
-    navigation.navigate(NavigatorName.AddAccounts, navigationParams);
+    if (llmNetworkBasedAddAccountFlow?.enabled)
+      navigation.navigate(NavigatorName.AddAccountsV2, navigationParams);
+    else navigation.navigate(NavigatorName.AddAccounts, navigationParams);
```

Comment:
i don't think we'll have win point in terme of algorithm complexity. In my implementation we have a first operation as `condition` + extracting the `enabled` prop from the `llmNetworkBasedAddAccountFlow `+ accessing the navigate call + the invoke of `navigate(..)`  -> 4 operations 

In your suggestion: it start with a accessing the navigate call + the invoke of `navigate(..)` + processing dynamic argument in a ternary condition  (here at build level it will be switched to a classical if/else) + getting the argument  -> 4 operations

Both don't have any affect on performance, one is more readable and have less cognitive complexity than the other. I randomly use both.

Reply:
It was only for readability purposes, I find that it's easier to understand that the condition only impacts the screen name but as you wish for this one.

Reply:
done ✔️